modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Pagination.tsx

```diff
@@ -132,6 +133,7 @@ function Pagination({
 				type: 'success',
 			});
 			setListOfItemsPerPage(itemsPerPage);
```

Comment:
Using the `updateFDSTabHandler` would make `setListOfItemsPerPage(itemsPerPage)` unneccesary

Reply:
Yep, I think about it but forgot to change and test. Thx